tests/dmctl_basic/check_list/check_task.sh

```diff
@@ -39,5 +39,6 @@ function check_task_error_database_config() {
     task_conf=$1
     run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
         "check-task $task_conf" \
-        "database driver error" 1
+        "Access denied for user" 1 \
+        "Please check the database config in configuration file"
```

Comment:
missing expected count for "Please check the database config in configuration file"?

Reply:
adress in 6c37e3d